bookkeeper-server/pom.xml

```diff
@@ -222,8 +222,21 @@
         <version>2.7.3</version>
         <scope>test</scope>
     </dependency>          
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-tcnative-boringssl-static</artifactId>
+      <version>${netty-boringssl.version}</version>
+      <classifier>${os.detected.classifier}</classifier>
```

Comment:
We should use netty-tcnative-boringssl-static without classifier which contains native bindings for every os